tests/rptest/services/tls.py

```diff
@@ -99,10 +99,15 @@ def _with_dir(self, name):
 
     def _exec(self, cmd):
         self._logger.info(f"Running command: {cmd}")
-        output = subprocess.check_output(cmd.split(),
-                                         cwd=self._dir.name,
-                                         stderr=subprocess.STDOUT)
-        self._logger.debug(output)
+        try:
+            output = subprocess.check_output(cmd.split(),
+                                             cwd=self._dir.name,
+                                             stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            self._logger.error(f"openssl error: {e.output}")
```

Comment:
> subprocess.CalledProcessError: Command '['openssl', 'genrsa', '-out', '/tmp/tmp3e6zdxjw/docker-rp-2.key', '2048']' returned non-zero exit status 1.

wow i never would have imagined. can't wait to see what the issue is. _can't do math today_, _not enough entropy_, ...